src/Sylius/Bundle/OrderBundle/Resources/config/services.xml

```diff
@@ -70,5 +70,10 @@
             <tag name="sylius.cart_context" priority="-777" />
         </service>
         <service id="sylius.context.cart" class="Sylius\Component\Order\Context\CompositeCartContext" />
+
+        <service id="sylius.remover.expired_cart" class="Sylius\Component\Order\Remover\ExpiredCartsRemover">
```

Comment:
sylius.expired_carts_remover
sylius.expired_orders_remover

I don't think it is worth creating a group of services for these:

1) We only have 2 of them here
2) There is no common "remover" concept in Sylius